src/main/java/org/cactoos/text/BytesAsText.java

```diff
@@ -114,4 +114,9 @@ public String asString() throws IOException {
         return new String(this.bytes.asBytes(), this.charset);
     }
 
+    @Override
+    public int compareTo(final Text text) {
+        return new UncheckedText(text).compareTo(this);
```

Comment:
@fabriciofx Maybe something like this `new UncheckedText(this).compareTo(text)` better?